__tests__/app.test.js

```diff
@@ -103,8 +103,45 @@ describe("GET /api/articles/:article_id", ()=>{
     })
     test("404: responds with an error message when passed a valid, but non existent article ID", ()=>{
         return request(app).get("/api/articles/123123").expect(404).then((response)=>{
-            expect(response.body).toEqual({message: "No article found"})
+            expect(response.body).toEqual({message: "404 Not Found"})
         })
     })
+})
 
+describe("GET /api/articles", ()=>{
+    test("200: responds with a status of 200", ()=>{
+        return request(app).get("/api/articles").expect(200).then((response)=>{
+            expect(response.status).toBe(200)
+        })
+    })
+    test("200: responds with all articles, with the following properties: author, title, article_id, topic, created_at, votes, article_img_url, comment_count", ()=>{
+        return request(app).get("/api/articles").then((response)=>{
+            const articles = response.body.articles
+            articles.forEach((article) => {
```

Comment:
Nice checking that each article has the properties that you expect, however you should check the length of the array you are getting back as if you sent an empty array back this would cause a false passing test.